src/emulator/storage/cloudFunctions.ts

```diff
@@ -69,6 +85,22 @@ export class StorageCloudFunctions {
       }, // bucket
       data: objectMetadataPayload,
     });
+    /** Cloud Event */
+    const ceAction = STORAGE_V2_ACTION_MAP[action];
```

Comment:
I think we have opportunity to make this a bit safer - either have a runtime check to make sure that ceAction isn't undefined or have a type-level checking to make sure that `STORAGE_V2_ACTION_MAP` mapping contains all the mapping for possible values of `StorageCloudFunctionAction`